dds/idl/be_global.h

```diff
@@ -173,19 +174,27 @@ class BE_GlobalData {
   bool root_default_nested() const;
 
   /**
-   * Check if a struct field has been declared a key.
+   * If node has the key annotation, this sets value to the key annotation
+   * value and returns true.
+   * If node does not have the key annotation, this sets value false and
```

Comment:
"sets value false" or "sets value to the default"?

Reply:
It should say "sets value to false".